doc/integrator/routing.rst

```diff
@@ -80,8 +81,8 @@ profile: string, url-path of the profile, example::
 If more than one profile are supplied, the component will show a drop-down-menu for the user to select a profile.
 
 
-searchDefaultParams
-^^^^^^^^^^^^^^^^^^^
+ngeoNominatimSearchDefaultParams
```

Comment:
For me the documentation about nominatim should be in ngeo, not in c2cgeoportal...

Reply:
It's a setting an integrator would care about when using the routing feature. For example if you don't restrict the country, you will get a lot of useless match from all around the world.